src/externs/crypto-js.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @fileoverview crypto-js library extern.
+ */
+
+/**
+ * @constructor
+ * @return {!CryptoJS}
+ */
+function CryptoJS() {}
+
+/**
+ * @param {string} message
+ * @return {string}
```

Comment:
No. It returns an object whose toString() gives a string.

On Sunday, March 20, 2016, Liana Yang <notifications@github.com> wrote:

> In src/externs/crypto-js.js
> <https://github.com/ViDA-NYU/genotet/pull/145#discussion_r56759401>:
>
> > @@ -0,0 +1,15 @@
> > +/**
> > + * @fileoverview crypto-js library extern.
> > + */
> > +
> > +/**
> > + * @constructor
> > + * @return {!CryptoJS}
> > + */
> > +function CryptoJS() {}
> > +
> > +/**
> > + * @param {string} message
> > + * @return {string}
>
> Shouldn't it return the result of hashed string???
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/ViDA-NYU/genotet/pull/145/files/83dd548aa3090df14c14b115859530f16cb2a2a5#r56759401>
>